daffodil-japi/src/main/scala/org/apache/daffodil/japi/Daffodil.scala

```diff
@@ -1155,3 +1156,23 @@ class DaffodilUnparseContentHandler private[japi] (sContentHandler: SDaffodilUnp
   override def skippedEntity(name: String): Unit =
     contentHandler.skippedEntity(name)
 }
+
+/**
+ * Returns the EntityResolver used by Daffodil to resolve import/include
+ * schemaLocations.
+ *
+ * DFDLCatalogResolver uses an org.apache.xml.resolver.Catalog/CatalogManager to
```

Comment:
Don't mention `DFDLCatalogResolver`, that class isn't public so mentioning it in public API is just going to be confusing. The user will just get an `EntityResolver`, `XmlEntityResolver`, etc.--that's all they need to know about.